src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -128,7 +128,11 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
 
             var lc = new LoggerConfiguration();
 
+            // apply inherited configuration except minimum level
+            // since parent logger can have overrides
             _applyInheritedConfiguration(lc);
+            lc.MinimumLevel.Is(LevelAlias.Minimum);
```

Comment:
Since `_applyInheritedConfiguration` ends up calling:

```csharp
        void ApplyInheritedConfiguration(LoggerConfiguration child)
        {
            if (_levelSwitch != null)
                child.MinimumLevel.ControlledBy(_levelSwitch);
            else
                child.MinimumLevel.Is(_minimumLevel);
        }
```

it seems like removing that call here might be a cleaner fix?

I had a quick attempt at figuring out the original purpose of `_applyInheritedConfiguration` and can't track it down; I suspect it may have been part of an earlier version that took advantage of `Logger` directly implementing `ILogEventSink.Emit()` (without any level check), but can't immediately tell from the source.

Removing it entirely seems not to break any tests. Am I missing anything?

Reply:
I've tracked it down to this [pr](https://github.com/serilog/serilog/pull/654). Part of it was the fix for [804](https://github.com/serilog/serilog/issues/804#issuecomment-231665651) issue.

The issue was about case with default minimum level of the root less than `Information` which results in explicit setting minimum for subloggers by users. 

The open question - why it was choosen the same as for the parent instead of `Verbose`..

Hint about `Logger` as `ILogEventSink` led me to such examples:

Behavior before this pr:
```csharp
new LoggerConfiguration()
    .MinimumLevel.Verbose()
    .WriteTo.Sink(
        new LoggerConfiguration()
            .MinimumLevel.Fatal()
            .WriteTo.Console()
            .CreateLogger())
    .CreateLogger()
    .Verbose("oops"); // surpasse the Fatal filter
```
but
```csharp
new LoggerConfiguration()
    .MinimumLevel.Verbose()
    .WriteTo.Sink(
        new LoggerConfiguration()
            .MinimumLevel.Fatal()
            .WriteTo.Logger(lc => lc.WriteTo.Console())
            .CreateLogger())
    .CreateLogger()
    .Verbose("oops"); // suppressed by the sublogger's Fatal filter
```

After this pr both examples will result in surpassing the Fatal filter. So it actually make the behavior consistent :)

So it makes sense to remove those lines as you pointed.